src/Backend/Core/Js/backend.js

```diff
@@ -1211,6 +1211,19 @@ jsBackend.forms =
         jsBackend.forms.tagsInput();
         jsBackend.forms.meta();
         jsBackend.forms.datePicker();
+        jsBackend.forms.bootstrapTabFormValidation();
+    },
+
+    bootstrapTabFormValidation: function () {
```

Comment:
What if there are multiple invalid fields?

Reply:
it will always go to the first invalid field it finds, if there are multiple you can still correct them or you go to that tab the next attempt to submit.